health/health_json.c

```diff
@@ -1,92 +1,129 @@
 // SPDX-License-Identifier: GPL-3.0-or-later
 
 #include "health.h"
+#include "libnetdata/libnetdata.h"
+
+#define BOOL_STR(x)         (x) ? "true" : "false";
+#define DEFAULT_STR(x, y)   (x) ? (x) : (y);
+
+static void
+kv_html_escape(BUFFER *wb, const char *key, const char *value,
```

Comment:
Sorry, I did not understand prefix `kv`, what does it mean in health context?

Reply:
If we are talking about the pair `key` and `value`, would it better to move it for `libnetdata/json` instead to keep a `kv` here inside `health` directory?

Reply:
You are right, `kv` stands for `key, value` because these set of functions accept a buffer, key, value, prefix and suffix (I'm open to suggestions if you find this overly succinct).

> would it better to move it for libnetdata/json instead to keep a kv here inside health directory?

I wouldn't mind doing so. I didn't already do so because we expect to replace this code soon with proper validation from `json-c` and we do not intend to have any other external users of these functions.

Reply:
Perfect, so I will give a suggestion you append `health_`, in front of `kv`, because `kv` is generic to be inside health, be free to accept or reject this idea. I called your attention for this, because we do not have any `kv` prefix inside our health code.

> I wouldn't mind doing so. I didn't already do so because we expect to replace this code soon with proper validation from json-c and we do not intend to have any other external users of these functions.

Considering this, let us keep inside health only.